targets/csharp/source/source/PlayFabGameServerHttp.cs

```diff
@@ -0,0 +1,72 @@
+﻿// GSDK Only has the following supported frameworks for now
+#if NETSTANDARD2_0 || NETCOREAPP2_1
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+
+namespace PlayFab
```

Comment:
None of these belong in the root PlayFab namespace.
I think all of this PR should be in PlayFab.GSDK, or something similar to indicate the specific use.